lrpar/src/lib/dijkstra.rs

```diff
@@ -66,8 +66,7 @@ where
         .drain(usize::from(c)..usize::from(c) + 1)
         .next()
         .unwrap();
-    while !scs_todo.is_empty() {
-        let n = scs_todo.pop().unwrap().1;
+    while let Some((_, n)) = scs_todo.pop() {
```

Comment:
Nice, Similar to this I was hoping to post a patch soon which looks at cases where tuple `.0` and `.1` etc are used and see if there are any cases where pattern matching could improve clarity.

Reply:
This is the only one I did by hand (but I wouldn't have thought to look for it without Clippy's other suggestions).